includes/polling/functions.inc.php

```diff
@@ -56,7 +56,7 @@ function poll_sensor($device, $class)
     $misc_sensors = array();
     $all_sensors = array();
 
-    foreach (dbFetchRows("SELECT * FROM `sensors` WHERE `sensor_class` = '?' AND `device_id` = ?", array($class, $device['device_id'])) as $sensor) {
+    foreach (dbFetchRows("SELECT * FROM `sensors` WHERE `sensor_class` = ? AND `device_id` = ?", array($class, $device['device_id'])) as $sensor) {
```

Comment:
Well, the code with the quoted '?' does give me problems. Then the hddtemp drop to 0 degrees in my graphs, and I have alerts for alle servers with unix-agent / hddtemp enabled. (temp = 0 C)